cmd/agentbasedinstaller/register.go

```diff
@@ -75,7 +75,21 @@ func RegisterCluster(ctx context.Context, log *log.Logger, bmInventory *client.A
 		return nil, errorutil.GetAssistedError(registerClusterErr)
 	}
 
-	log.Info("Registered cluster with id: " + clusterResult.Payload.ID.String())
+	log.Infof("Registered cluster with id: %s", clusterResult.Payload.ID)
+
+	annotations := aci.GetAnnotations()
+	if installConfigOverrides, ok := annotations[controllers.InstallConfigOverrides]; ok {
+		updateInstallConfigParams := &installer.V2UpdateClusterInstallConfigParams{
+			ClusterID:           *clusterResult.Payload.ID,
+			InstallConfigParams: installConfigOverrides,
+		}
+		_, updateClusterErr := bmInventory.Installer.V2UpdateClusterInstallConfig(ctx, updateInstallConfigParams)
```

Comment:
I suspect that ignoring the return value here is the reason the new test is failing.